### DISCUSSION TITLE
en/diary/march-2024/

### DISCUSSION THREAD

**Discussion by giscus** [Dagwbl/dagwbl.github.io#2] [Announcements]
# en/diary/march-2024/

Mar 28 There are two assignments as following:
There are a number of tests available which can suggest if a person is telling the truth, but knowing which ones are accurate is not easy. A newly created test is claimed to be the most accurate yet in lie detection.However, questions have been raised about its accuracy and ethics.
Students who wish to take a break from their studies will need to put in an application for Leave of Absence.

https://dagwbl.github.io/en/diary/march-2024/

**Comment by Dagwbl** [comment]
Whew！
